### DISCUSSION TITLE
fun

### DISCUSSION THREAD

**Discussion by OndrejMottl** [CUNI-SPRouT/SPROuT-Forum#6] [Casual chat]
# LOOK!

This is a **coool** sentence [this is a link](www.google.com)

**Comment by PavelKucera124** [comment]
***So I Try to write text strong and italic***

**Comment by al-YangG** [comment]
<ins> Underline text </ins>

**Comment by HelenaPijalkova** [comment]
That is a *veery* cool sentence, [nice](https://en.wikipedia.org/wiki/File:Hoverflies_mating_midair.jpg)

**Comment by KadyanSonia** [comment]
## Header 2

### Header 3

*italic*
[My-link] (https://ondrejmottl.github.io/collaboration/code_convention.html)

**Comment by SvandaPetr** [comment]
# header
## smaller header
### even smaller header
#### even more *SMALLER* header?
##### how far this goes?
###### ok..?
####### this is the end...

**Comment by JakubNovak0204** [comment]
# Mark down text

"original" *cool* **sentence**

**Comment by learningYC** [comment]
# A title
**okay**
## Subtile
*cool*
---------------------------------------
[**To the chat*](https://github.com/orgs/CUNI-SPRouT/discussions/6)

**Comment by sayantika91** [comment]
# Title 1 
## Subtitle 2 
### Subtitle 3
#### Subtitle 4 
##### Subtitle 5 
*Sayantika* 
~~I am dead~~

**Comment by KaterinaKasparova** [comment]
**sentence** *without* any >sense

**Comment by LucieHolzbachova** [comment]
*Hola*  **Markdown**
